posts/2019/04/whats-new-in-sympy-14.md

```diff
@@ -58,18 +58,18 @@ do the release itself.
 Prior to SymPy 1.4, SymPy expressions in the notebook rendered by default with their
  string representation. To get `LaTeX` output, you had to call `init_printing()`:
 
-![SymPy 1.3 rendering in the Jupyter lab notebook](/images/sympy-1.3-notebook.png)
+![SymPy 1.3 rendering in the Jupyter lab notebook ](/images/sympy-1.3-notebook.png)
 
 In SymPy 1.4, SymPy expressions now automatically render as LaTeX in the notebook:
 
-![SymPy 1.4 rendering in the Jupyter lab notebook](/images/sympy-1.4-notebook.png)
+![SymPy 1.4 rendering in the Jupyter lab notebook ](/images/sympy-1.4-notebook.png)
 
 However, this only applies automatically if the type of an object is a SymPy
 expression. For built-in types such as lists or ints, `init_printing()` is
 still required to get LaTeX printing. For example, `solve()` returns a list,
 so does not render as LaTeX unless `init_printing()` is called:
 
-![SymPy 1.4 rendering in the Jupyter lab notebook with init_printing()](/images/sympy-1.4-notebook-2.png)
+![SymPy 1.4 rendering in the Jupyter lab notebook with init_printing() ](/images/sympy-1.4-notebook-2.png)
```

Comment:
Code snippet in an interactive Jupyter session showing the usage of init_printing for list objects